Makefile

```diff
@@ -57,23 +57,32 @@ else
 	@$(REBAR) eunit
 endif
 
+.dialyzer_plt:
```

Comment:
should we add a rebar.config hook to clean this when cleaning?

Reply:
Sure, if it makes sense for you. I just tweaked the `rebar 2` "options" to make what was there work properly, not because I'm using rebar 2 still. Do you want me to add that "clean" statement?

Reply:
Actually, I'm going to go ahead and assume you want the change, so I'll perform it; otherwise you wouldn't have asked the question :)

Reply:
Actually, now that I read your question properly, you talk about a _hook_. I'll have to look up how to do that in `rebar 2`, since I don't think I've ever done it previously...

Reply:
I did it using a "post" hook. Though it works for `rebar3` also it'll have no affect since (to my knowledge) the PLT is usually generated inside `_build`.

Reply:
yeah, I think I got the 2 rebars confused, probably was fine to leave as-is, but it's certainly harmless. I can just merge as-is.